apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1536,6 +1536,17 @@
                     "type" : "boolean",
                     "default" : true
                 },
+                "genesis_accounts" : {
+                    "description" : "Pre-funded accounts added to the genesis state trees",
```

Comment:
Could you please specific the overlapping implementation we discussed in the chat here. (Merge+sum).

Reply:
Fixed.